warp10/src/main/java/io/warp10/script/functions/TYPEOF.java

```diff
@@ -175,7 +178,7 @@ public static String typeof(Class c) {
       return TYPE_MATCHER;
     } else if (WarpScriptStack.Mark.class.isAssignableFrom(c)) {
       return TYPE_MARK;
-    } else if (Key.class.isAssignableFrom(c)) {
+    } else if (Key.class.isAssignableFrom(c) || PGPSecretKeyRing.class.isAssignableFrom(c) || PGPPublicKey.class.isAssignableFrom(c)) {
```

Comment:
```
'-----BEGIN%20PGP%20PUBLIC%20KEY%20BLOCK-----%0AVersion%3A%20PGP%202.6.2%0A%0AmQCNAiv8ZoAAAAEEAKc4d45hN5qFM79nWGLkrGWputWmtdxJk0BZEbi0kNRbJBC2%0Ap10ASImd%2F%2FcCDwLR2alBUSt8O2WGik9PBZgthjMOenoDmzKiG8BkE9AFKonyxvD2%0AlDnqbydXi%2BYQmOTsWSw4jTTSb3cflhVkf8hVUVpMFQThafV0CmV5hLjHqWbdAAUR%0AtCJQaGlsaXAgUi4gWmltbWVybWFubiA8cHJ6QGFjbS5vcmc%2BiQCVAgUQLB48XPTK%0AAIGN5yLZAQHWtwP%2FRBiLPN4dnt8sm9qZtK0HPYV0hfdZ4IiSfR0V52uKKMQsIrBJ%0Ax2c5Z2vurBLeKkh8Oecf%2FX%2BZh2mEenrymR%2FurBCf8xGQnyTPew4t%2F3IQ5KXsqi2b%0AuOTysk9Pkk%2BcqxZTEXJQWixB3fVKrCkR02xbWcRXQ%2FpPs0ObOE4VLtQT1G2JAGAC%0ABRAsHUvayj2h%2FYZap%2FMBAcRRAliFU3EgFQdOGkC2nCupBAitm%2FR6CkKtayTZHQrd%0Ae5I%2FpCY0bGUc6alEnSuqZUyA9HC1fMg0iqsqM4vrQ6PZr2qV0IXDY4miyYNRJyKJ%0AAJUCBRAr%2FGdG4nXeDv9n9wsBAYr%2FBACh4r3DZsq9IOy2mcd%2B0D0qKV0Ymb887Lw0%0AKjIBKLmUIqXUw2Rbn8bAnr%2BGF0qGE61oAHdSSN7pLROmfjl7nMVaODXHOxYuLHAn%0AHZ0Z%2FAQsLc5U055Pm6rQ5lKrSaL4z9mEKJTWM9hQDYMZpH4oAC8hIz%2BapYH%2B%2FhBj%0AZR9w7AKlu4kAlQIFEC5jaYfFLUdtDb%2BQbQEBOKED%2FiX0D19aSu8cwbBWloKt6Tij%0A9VUtCVJU5CAilrQZRs5h%2Fp0YCFvoMUIv8JtOgK2KGWsUuyRF2kMgZ4X4n6n6OWmJ%0A3VMizh7bTXm1IWXcRx98J2vzHkleoHTMKdc0X7wiXQXJelzyE34KVxpQHaMSyiZq%0AwClzty5Lt%2FtMC37THTDFiQBVAgUQLeTVN1UFZvpNDE7hAQEspgH%2BL0jcCteJhBW3%0ANwJaXygzDP4Lq%2FJNk2X7Pgy5wGPstlBZ58EMkm%2BmgP5PZTG%2Fpx3DyFsMcuBQP%2BxZ%0AfrxWge1714kAlQIFEC3LGYtleYS4x6lm3QEBuTMD%2FiaTje7mA8B2hVs8PDM1Z2nh%0AOd2oMHwGJI4fy8H7SQSVPHV3BlRZpkIou0%2FvbEZY9b8WlUm9QEETWZEMmKYgHZTb%0AEzkkFtq0zNqgfsuORoe262c%2FpbYofNrMmnYok7K3vIAoqbUV6JlRRJ0oo3y%2B8IMi%0AfjnNZKfgNpGmJDCMTh8M%0A%3DHpg1%0A-----END%20PGP%20PUBLIC%20KEY%20BLOCK-----'
PGPRING
0 GET TYPEOF

```

still returns a raw class name and not KEY.

Reply:
Added type KEYRING